README.md

```diff
@@ -37,27 +50,31 @@ We test this foreign data wrapper on Ubuntu Server 18.04 using PostgreSQL-11 tog
   sudo ldconfig
   ```
 
-- Build this foreign data wrapper with RocksDB:
+- Build this foreign data wrapper:
   
   ```sh
+  git clone https://github.com/vidardb/PostgresForeignDataWrapper.git
+
   cd PostgresForeignDataWrapper 
 
   make
 
   sudo make install
   ```
 
-- Build this foreign data wrapper with VidarDB:
+  If you want to build this foreign data wrapper for [VidarDB](https://github.com/vidardb/vidardb), you can build like this:
 
   ```sh
-  cd PostgresForeignDataWrapper 
-
   make VIDARDB=true
+  ```
 
-  sudo make install
+- Instead of using this foreign data wrapper you need to add it to ```shared_preload_libraries``` in your ```postgresql.conf```:
```

Comment:
Use \`shared_preload_libraries\` and \`postgresql.conf\` instead of \`\`\`shared_preload_libraries\`\`\` and \`\`\`postgresql.conf\`\`\`.

`xxx` is for in-line code, while 

```
xxx
```
is for code block.

Reply:
What `Instead of using this foreign data wrapper` means? You want to express `Before using the foreign data wrapper`?

Reply:
Sorry, I misunderstood. I have updated now and please review again.